_meta/config/common.reference.p2.yml.tmpl

```diff
@@ -121,6 +121,12 @@ inputs:
 #   # Default is false
 #   exponential: false
 
+# agent.limits:
+#   # limits the number of operating system threads that can execute user-level Go code simultaneously.
+#   # Translates the into GOMAXPROCS environment variable for each Go process started by the agent.
```

Comment:
This should apply to the agent itself as well.

Reply:
Good point, I'll add it.